scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -62,35 +67,49 @@ abstract class ScanResultsStorage {
          * [FileBasedStorage] using a [XZCompressedLocalFileStorage] as backend is configured.
          */
         fun configure(config: ScannerConfiguration) {
-            val configuredStorages = listOfNotNull(
-                config.fileBasedStorage,
-                config.postgresStorage
-            )
+            storage = if (config.storages.isNullOrEmpty()) createDefaultStorage()
+            else createCompositeStorage(config)
+            log.info("ScanResultStorage has been configured to ${storage.name}.")
```

Comment:
Please always use the syntax with curly braces, as that avoid string expansion if the string should not be logged:

                log.info { "ScanResultStorage has been configured to ${storage.name}." }